src/ContentSecurityPolicy/DirectiveSet.php

```diff
@@ -15,12 +15,12 @@
 
 class DirectiveSet
 {
-    const TYPE_SRC_LIST = 'source-list';
-    const TYPE_SRC_LIST_NOFB = 'source-list-no-fallback';
-    const TYPE_MEDIA_TYPE_LIST = 'media-type-list';
-    const TYPE_ANCESTOR_SRC_LIST = 'ancestor-source-list';
-    const TYPE_URI_REFERENCE = 'uri-reference';
-    const TYPE_NO_VALUE = 'no-value';
+    public const TYPE_SRC_LIST = 'source-list';
+    public const TYPE_SRC_LIST_NOFB = 'source-list-no-fallback';
+    public const TYPE_MEDIA_TYPE_LIST = 'media-type-list';
+    public const TYPE_ANCESTOR_SRC_LIST = 'ancestor-source-list';
+    public const TYPE_URI_REFERENCE = 'uri-reference';
+    public const TYPE_NO_VALUE = 'no-value';
```

Comment:
I'd argue all these should be private, especially as you're going for a 3.0 it's worth really thinking about visibility of things instead of defaulting all constants to public just to add accessors.

Reply:
👍 I'm going to create an issue with things I think we have to do before 3.0, one of them is the visibility in general, having no accessor meant to have it public, this is just making it explicit (it was added by php-cs-fixer).